### PR TITLE
clarify public domaing licenses

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

**Review by RubenRBS — DISMISSED** [review]

File: docs/adding_packages/conanfile_attributes.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The mandatory license attribute of each recipe **should** be a [SPDX license](ht
 
 Where the SPDX guidelines do not apply, packages should do the following:
 
-* When no license is provided or it's under the ["public domain"](https://fairuse.stanford.edu/overview/public-domain/welcome/) - these are not a license by itself. Thus, we have [equivalent licenses](https://en.wikipedia.org/wiki/Public-domain-equivalent_license) that should be used instead. If a project falls under these criteria it should be identified as the [Unlicense](https://spdx.org/licenses/Unlicense) license.
+* When no license is provided or it's under the ["public domain"](https://fairuse.stanford.edu/overview/public-domain/welcome/) - these are not a license by itself. The author original intent and files should be respected, and the relevant files defining the "public domain" status should be the ones packaged in the "licenses" folder, for example the source repository README. The ``license`` field should be as close as the original one as possible, like ``license = "Public-domain"``.
```
Comment (jcar87) [inline docs/adding_packages/conanfile_attributes.md:56]: I would clarify here:

> When no license is provided

If the authors of an open source library don't specify any license terms, erring on the side of caution we cannot infer any, and disallow this from Conan Center altogether. The rest of the text looks good to me: the intent of the original author, including the exact wording, should be respected.

Comment (prince-chrismc) [reply docs/adding_packages/conanfile_attributes.md:56]: There's a contradiction between the last line of this and the next one

> the original one as possible, like ``license = "Public-domain"``.

vs

> the value should be set to `LicenseRef-` as a prefix, followed by the name of the file which contains the custom license.

I do not see why listing the file with the custom license could not be used in all cases 🤔

Comment (prince-chrismc) [reply docs/adding_packages/conanfile_attributes.md:56]: I think this is excellent improvement 👏

**Review by RubenRBS — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by prince-chrismc** [comment]
This was not finished. RIP

**Comment by SSE4** [comment]
> This was not finished. RIP

what do you mean?

**Comment by prince-chrismc** [comment]
The PR was never updated with the discussion https://github.com/conan-io/conan-center-index/pull/18481#discussion_r1260070682

**Pull request merged**